tests/attr/layer/test_grad_cam.py

```diff
@@ -25,6 +25,33 @@ def test_simple_input_conv_relu(self):
         inp = 1.0 * torch.arange(16).view(1, 1, 4, 4).type(torch.FloatTensor)
         self._grad_cam_test_assert(net, net.relu1, inp, [[0.0, 4.0], [28.0, 32.5]])
 
+    def test_simple_input_conv_without_final_relu(self):
+        net = BasicModel_ConvNet_One_Conv()
+        inp = 1.0 * torch.arange(16).view(1, 1, 4, 4).type(torch.FloatTensor)
```

Comment:
nit: we could also do `inp.float()` instead of `type(torch.FloatTensor)`?
This applies also to other tests. Actually, do we need to multiply with `1.0` ?